groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy

```diff
@@ -0,0 +1,37 @@
+package com.powsybl.tutorials.groovy_modifications
+
+import com.powsybl.iidm.modification.topology.CreateBranchFeederBaysBuilder
```

Comment:
All these imports could be hidden and done by script loader

Reply:
Do you think we should add a script loader now or wait for the wrapper we intend to add in core (which would change the scripts I assume)?

Reply:
What we wanted was a script example that can be executed from the iTools `run-script`